pkg/README.md

```diff
@@ -0,0 +1,30 @@
+# Protobuf packaging
+
+This directory contains Bazel rules for building packaging and distribution
+artifacts.
+
+*Everything in this directory should be considered experimental.*
+
+## Protocol compiler binary packaging
+
+The protocol compiler is used in binary form in various places. There are rules
+which package it, along with commonly used `.proto` files, for distribution.
+
+## Source distribution packaging
+
+Protobuf releases include source distributions, sliced by target language (C++,
+Java, etc.). There are rules in this package to define those source archives.
+These depend upon `pkg_files` rules elsewhere in the repo to get the contents.
+
+The source distribution files should include the outputs from `autogen.sh`, but
+this isn't something we can reliably do from Bazel. To produce fully functioning
+source distributions, run `autogen.sh` before building the archives (this
```

Comment:
Is this a long-term necessity or a short-term workaround?

Reply:
Probably long term... at least, as long as we want the distributions to be buildable with autotools.

Having toyed in the past with trying to integrate autotools with Starlark rules, and given the complexity of our current autotools logic, I don't think it's currently worth trying to do as part of the build. There are a lot of ways to change that assessment, though.